packs/data/feature-effects.db/effect-aquatic-combat-no-swim-speed.json

```diff
@@ -0,0 +1,85 @@
+{
+    "_id": "ZwWqIFU50EY7vl2l",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p>Use these rules for battles in water or underwater:</p>\n<ul>\n<li>You're @Compendium[pf2e.conditionitems.Flat-Footed]{Flat-Footed} unless you have a swim Speed.</li>\n<li>You gain resistance 5 to acid and fire.</li>\n<li>You take a -2 circumstance penalty to melee slashing or bludgeoning attacks that pass through water.</li>\n<li>Ranged attacks that deal bludgeoning or slashing damage automatically miss if the attacker or target is underwater, and piercing ranged attacks made by an underwater creature or against an underwater target have their range increments halved.</li>\n<li>You can't cast fire spells or use actions with the fire trait underwater.</li>\n<li>At the GM's discretion, some ground-based actions might not work underwater or while floating.</li>\n</ul>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "Note",
+                "predicate": {
+                    "all": [
+                        "fire"
+                    ]
+                },
+                "selector": "all",
+                "text": "<p>You can't cast fire spells or use actions with the fire trait underwater.</p>"
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": {
+                    "any": [
+                        "slashing",
+                        "bludgeoning"
+                    ]
```

Comment:
@stwlam is it possible to predicate on the damage type a strike would make on an attack roll? These don't work atm, but the modifier is there as a toggle on every strike

Reply:
"weapon:damage:type:slashing"